auditlog/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Audit Log",
-    "version": "18.0.1.0.0",
+    "version": "18.0.1.0.1",
```

Comment:
Liever nu niet het versienummer bumpen. De module is bij ons nog niet gedeployed, en deze versie nu ophogen zal in de toekomst misschien een OCA bump naar hetzelfde nummer maskeren.